### PR TITLE
Home page.

### PR DESCRIPTION
**Pull request by sviastyn**

### CODE REVIEW

File: src/TechMarket.WEB/tech-market/src/components/pages/home/Home.jsx [hunk 1]
```diff
@@ -1,9 +1,47 @@
...
+            <Container className='mt-3'>
+                <Row className='justify-content-between'>
+                    <Col sm={12} md={2} lg={3} className='home_col'>
+                        <h4>Title#1</h4>
+                        <ul>
+                            <li>Lorem</li>
+                            <li>Lorem, ipsum dolor.</li>
+                            <li>Lorem ipsum dolor sit amet.</li>
+                            <li>Lorem ipsum dolor, sit amet consectetur.</li>
+                        </ul>
```
Comment (Yurii-Borys) [inline src/TechMarket.WEB/tech-market/src/components/pages/home/Home.jsx:24]: If you use <li>, they must have unique key

Comment (Yurii-Borys) [reply src/TechMarket.WEB/tech-market/src/components/pages/home/Home.jsx:24]: *li

File: src/TechMarket.WEB/tech-market/src/components/Slider/ImageSlider.jsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import React, { useState } from 'react'
-import "./ImageSlider.css"
+import './ImageSlider.css'
+import { ArrowRightCircle, ArrowLeftCircle } from 'react-bootstrap-icons';
+
```
Comment (BodyDuk) [inline src/TechMarket.WEB/tech-market/src/components/Slider/ImageSlider.jsx:4]: Extra line.

File: src/TechMarket.WEB/tech-market/src/components/footer/Footer.scss [hunk 3]
```diff
@@ -3,7 +3,6 @@
 .footer_dark {
```
Comment (BodyDuk) [inline src/TechMarket.WEB/tech-market/src/components/footer/Footer.scss:3]: Please correct the footer or revert the change.

File: src/TechMarket.WEB/tech-market/src/components/pages/home/Home.jsx [hunk 4]
```diff
@@ -1,9 +1,47 @@
 import React from 'react';
+import ImageSlider from './../../Slider/ImageSlider';
+import useGetSliderImages from './../../../api/hooks/getSliderImages';
+import PropTypes from 'prop-types';
+import { Container, Col, Row } from 'react-bootstrap';
+import './Home.css';
```
Comment (BodyDuk) [inline src/TechMarket.WEB/tech-market/src/components/pages/home/Home.jsx:6]: Please prioritase imports.

File: src/TechMarket.WEB/tech-market/src/components/pages/home/Home.jsx [hunk 5]
```diff
@@ -1,9 +1,47 @@
...
 
 const Home = () => {
+
+    let imagesData = useGetSliderImages().sliders.map(x => x.imageUrl);
```
Comment (BodyDuk) [inline src/TechMarket.WEB/tech-market/src/components/pages/home/Home.jsx:10]: Please use more meaningful names.

**Review by BodyDuk — CHANGES_REQUESTED** [review]

File: src/TechMarket.WEB/tech-market/src/components/Slider/ImageSlider.jsx [hunk 6]
```diff
@@ -12,16 +13,16 @@ const ImageSlider = ({ images }) => {
 
     const getPrev = () => {
         setCurrent(current === 0 ? length - 1 : current - 1);
-    };
+    };
```
Comment (BodyDuk) [inline src/TechMarket.WEB/tech-market/src/components/Slider/ImageSlider.jsx:16]: Extra space.

File: src/TechMarket.WEB/tech-market/src/components/pages/home/Home.css [hunk 7]
```diff
@@ -0,0 +1,9 @@
+.home_col {
+    border-radius: 20px;
+    box-shadow: .5rem .5rem 1rem rgb(223, 222, 222);
+}
+
+h4 {
+    display: flex;
+    justify-content: center;
+}
```
Comment (BodyDuk) [inline src/TechMarket.WEB/tech-market/src/components/pages/home/Home.css:9]: You can use bootstrap!

File: src/TechMarket.WEB/tech-market/src/components/pages/home/Home.jsx [hunk 8]
```diff
@@ -1,9 +1,47 @@
...
+                    <Col sm={12} md={2} lg={3} className='home_col'>
+                        <h4>Title#1</h4>
+                        <ul>
+                            <li>Lorem</li>
+                            <li>Lorem, ipsum dolor.</li>
+                            <li>Lorem ipsum dolor sit amet.</li>
+                            <li>Lorem ipsum dolor, sit amet consectetur.</li>
+                        </ul>
+                    </Col >
+                    <Col sm={12} md={4} lg={4} className='home_col'>
```
Comment (BodyDuk) [inline src/TechMarket.WEB/tech-market/src/components/pages/home/Home.jsx:26]: This breakpoint is unnecessary.

**Review by BodyDuk — CHANGES_REQUESTED** [review]

**Review by BodyDuk — APPROVED** [review]

**Pull request merged**